test/release/examples/primers/FFTWlib.chpl

```diff
@@ -195,73 +195,73 @@ proc runtest(param ndim : int, fn : string) {
       // Define domains to extract the real and imaginary parts for
       // in-place transforms
       //
-      reD = rD[..,..,0..(2*ldim-1) by 2]; 
-      imD = rD[..,..,1..(2*ldim-1) by 2]; 
+      reD = rD[..,..,0..(2*ldim-1) by 2];
+      imD = rD[..,..,1..(2*ldim-1) by 2];
     }
   }
 
   // FFTW does not normalize inverse transforms, so just compute the
   // normalization constant.
 
-  var norm = * reduce dims;
+  var norm =   reduce dims;
```

Comment:
I think your `*` removal command was too aggressive. You're changing actual code here.

Why do you need to remove `*` in the first place. It seems like a shame to have to change all the comments